tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -90,7 +90,14 @@ export class CardFobControllerComponent {
     return `translate(${this.endStepAxisPosition}px, 0px)`;
   }
 
-  startDrag(fob: Fob, affordance: TimeSelectionAffordance) {
+  stopEventPropagation(e: Event) {
+    if (e.stopPropagation) e.stopPropagation();
+    if (e.preventDefault) e.preventDefault();
+    e.cancelBubble = true;
```

Comment:
Is cancelBubble needed? I saw it is marked as deprecated. What is the reason for adding this in addition to stopPropagation and preventDefault?
https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelBubble

I'm having hard time reproducing the first glitch so not totally sure what would happen if we remove line 96. I will leave this decision to you.

Reply:
From what I was reading some old browsers do not have preventDefault or stopPropagation and cancelBubble is the only way to do it. Honestly I think it is only IE. I was back and forth on having it in there or not I do not feel strongly either way.

Reply:
You can search the rest of the TensorBoard codebase to see the assumptions we make about these methods and properties. You don't need cancelBubble. You can also assume stopPropagation and preventDefault exist. So the code could just be:

```
e.stopPropagation();
e.preventDefault();
```

It would be worth sending a cleanup PR to avoid confusion in the future.

Reply:
Will do. Thanks Brian.